Source/SwiftLintFramework/Models/Command.swift

```diff
@@ -27,6 +30,8 @@ public struct Command: Equatable {
         case this
         /// The command should only apply to the line following its definition.
         case next
+        /// The modifier string was invalid
```

Comment:
Nit: In documentation, prefered style is a period at the end of sentences/descriptions.